### PR TITLE
feature: support version command for karmadactl

### LINKED ISSUE #264: karmadactl should add version command
**What would you like to be added**:
Currently, we can't check `karmadactl` version that's the necessary information for such as debug.
```
-bash-4.2# karmadactl version
Error: unknown command "version" for "karmadactl"
Run 'karmadactl --help' for usage.
unknown command "version" for "karmadactl"
```
**Why is this needed**:

### PR DESCRIPTION
**Pull request by pingstar**
**What type of PR is this?**
> /kind feature

**What this PR does / why we need it**:
Support `karmadactl` version command that's can print some version information for debug like `kubectl version`.

**Which issue(s) this PR fixes**:
Fixes #264

### CODE REVIEW

File: pkg/karmadactl/version.go [hunk 1]
```diff
@@ -0,0 +1,32 @@
+package karmadactl
+
+import (
+	"fmt"
+	"io"
+
+	"github.com/karmada-io/karmada/pkg/version"
+	"github.com/spf13/cobra"
+)
```
Comment (RainbowMango) [inline pkg/karmadactl/version.go:9]: Could you please group the import by `std` --> `third-party` -->`karmada`?

**Review by RainbowMango — COMMENTED** [review]
Looks awesome! Thanks!

File: pkg/karmadactl/version.go (hunk 1 above)
Comment (pingstar) [reply pkg/karmadactl/version.go:9]: > Could you please group the import by `std` --> `third-party` -->`karmada`?

Of course, thank you for reviewing.

Comment (RainbowMango) [reply pkg/karmadactl/version.go:9]: Thanks, just a reminder, you need to sign the DCO[https://github.com/apps/dco] in your commit log. :)

**Pull request closed without merging**